accepted/global-json-updates.md

```diff
@@ -0,0 +1,790 @@
+  * If no matching SDK version in the feature band (xyz in x.y.z.nn) is found, the command being called fails.
+  * Each `global.json` must later be updated to use the new stable SDK.
+  * Due to Rule #2, there is no roll forward from preview to stable if they are both on the machine.
+    * The impact of this is mitigated by work we did in .NET Core 3.0 Preview 3 to replace earlier patch versions.
+* User wants to always use a specific SDK patch version, and receive an error if it is not on the machine.
+  * The user cannot currently do this.
+* User wants to always run the latest patch of a particular SDK feature band.
+  * Due to Rule #2, the only way to do this is to remove the previous version.
+    * The impact of this is mitigated by work we did in .NET Core 3.0 Preview 3 to replace earlier patch versions.
+* User includes a feature that requires an SDK that corresponds to a SDK feature band or higher, and wants to run the best available tools.
+  * There is no mechanism to require at least a specific version, but otherwise use the latest, or the latest in a range.
+  * Upgrading to a newer SDK will provide no benefit to projects tied to an earlier version.
+* User wants to know what version of the SDK is being used.
+  * `global.json` is a subtle, hidden and easily forgotten marker – programmers may not understand the downsides of leaving a global.json that was suggested in a blog post in place – or they may forget it.
+* User rarely wants the risk of running the unsupported scenario of multiple SDK versions in a solution build.
+  * There is nothing that encourages users to place `global.json` in the repository root or the same level as the solution. Using different versions of the SDK for different projects in the solution is not a supported scenario.
+* Hosted CI, including Azure DevOps and Kudu, need to provide a small number of SDKs.
+  * If there is no patch of the major/minor/feature version (xyz in `x.y.znn`) present, the build fails, causing friction between user and host needs.
+
+## Proposal C
+
+This proposal parallels [runtime roll-forward behavior as initially described](https://github.com/dotnet/core-setup/blob/master/Documentation/design-docs/roll-forward-on-no-candidate-fx.md)} and [recent improvements](https://github.com/dotnet/designs/blob/master/accepted/runtime-binding.md). All of the features proposed here may not be included in Phase 1. It will include at least `ignorePreview` and `"rollforward": "disable"`
+
+This is called Proposal C to differentiate it from two earlier proposals that were discarded but are included at the end of this document for reference.
+
+### Details
+
+The runtime options allow for three basic approaches: find nearest, find latest and find exact. 
+
+To avoid a backward compatibility issue, the default recovery strategy when a match is not found needs to match the old `dotnet` behavior. This defines the defaults, and the default behavior differs between runtime. 
```

Comment:
Is that required? Or put another way, are there some behaviors that are more important to maintain and others that are more borderline? For example, the [behavior where a lower version of the SDK is used than requested](https://github.com/dotnet/designs/pull/69/files#diff-52992d6b81ad6f9528f44e8bd5f44910R37) seems like a break we might be able to accept.